entities/storobj/buffer_pool.go

```diff
@@ -0,0 +1,31 @@
+package storobj
+
+import "sync"
+
+func newBufferPool(initialSize int) *bufferPool {
+	return &bufferPool{
+		pool: sync.Pool{
+			New: func() any {
+				return make([]byte, initialSize)
```

Comment:
since slice's length is reset on putting it back to pool, for consistency it could be created as 0 length (`make([]byte, 0, initialSize)`)

Reply:
Great point, let me add that.